libs/features/pages/home/src/lib/main/home/home-page.component.html

```diff
@@ -1,6 +1 @@
-<elewa-website-main-page>
-  <div mainPage class="home-page">
-    <elewa-website-projects-section></elewa-website-projects-section>
-    <elewa-website-news-section></elewa-website-news-section>
-  </div>
-</elewa-website-main-page>
```

Comment:
This is important, Your component should just be added in its correct placement

Reply:
@LemmyMwaura ##resolved